sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -13,6 +12,11 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/containerd/containerd/reference"
+	"github.com/opencontainers/go-digest"
+
+	"fmt"
```

Comment:
nit: Import grouping got messed up from std, third party, local, but not a critical issue.

Reply:
Weird that `goimports` doesn't re-arrange. I think since it got added to it's own block, goimports is treating it seperatly:

```
❯ find . -name "*.go" -not -path "*/vendor/*" | xargs goimports -w
```

so moving `fmt` into the top block with the other standard library things and running goimports to get it sorted should fix it.

Reply:
I actually had to run `goimports` to placate the linter and this is what the tool gave me.